spec/models/manageiq/providers/nuage/network_manager/event_catcher/messaging_handler_spec.rb

```diff
@@ -1,4 +1,6 @@
-describe ManageIQ::Providers::Nuage::NetworkManager::EventCatcher::MessagingHandler do
+# Note, we're using string class name to avoid autoloading the class since it requires
+# qpid_proton installed.
+describe "ManageIQ::Providers::Nuage::NetworkManager::EventCatcher::MessagingHandler", :qpid_proton => true do
```

Comment:
Actually seems like using strings here is causing the `described_class` below to fail

Reply:
just pushed a fix to not used `described_class` in the specs